Tests/Api/DocumentsControllerTests.cs

```diff
@@ -0,0 +1,65 @@
+using Api.Controllers;
+using Application;
+using Application.Services;
+using Core;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Tests;
+
+public class DocumentsControllerTests
+{
+    private readonly Mock<IInnerCircleHttpClient> _httpClientMock = new();
+    private readonly Mock<IPayslipsValidator> _payslipsValidatorMock = new();
+    private readonly IPayslipsValidator _validator = new PayslipsValidator();
+    private readonly ILogger<DocumentsController> _logger;
+    private readonly DocumentsController _controller;
+
+    public DocumentsControllerTests()
+    {
+        var loggerFactory = LoggerFactory.Create(builder => builder.AddConsole());
+        _logger = loggerFactory.CreateLogger<DocumentsController>();
+
+        _controller = new DocumentsController(_httpClientMock.Object, _payslipsValidatorMock.Object, _logger);
+    }
+
+    [Fact]
+    public async Task SendMailingPayslips_WhenEmailSenderServiceIsNotAvailable_CatchException()
+    {
+        _httpClientMock
+            .Setup(x => x.SendMailingPayslips(It.IsAny<List<PayslipsItem>>(), It.IsAny<List<Employee>>()))
+            .ThrowsAsync(new Exception());
+
+        var exception = await Assert.ThrowsAsync<Exception>(() => _controller.SendMailingPayslips(It.IsAny<List<PayslipsItem>>()));
```

Comment:
You can make this code cleaner. The records like "It.IsAny<List<PayslipsItem>>()" set into the private field. For example:
```
 private readonly List<PayslipsItem> _payslips = It.IsAny<List<PayslipsItem>>();
 private readonly List<Employee> _employees = It.IsAny<List<Employee>>();

_httpClientMock
    .Setup(x => x.SendMailingPayslips(_payslips, _employees))
    .ThrowsAsync(new Exception());

var exception = await Assert.ThrowsAsync<Exception>(() => _controller.SendMailingPayslips(_payslips));
``` 

And you should do it for all test methods